include/gba/m4a_internal.h

```diff
@@ -270,8 +297,8 @@ struct MusicPlayerTrack
     u8 lfoDelay;
     u8 lfoDelayC;
     u8 priority;
-    u8 echoVolume;
-    u8 echoLength;
+    u8 pseudoEchoVolume;
+    u8 pseudoEchoLength;
```

Comment:
What is the distinction here between `echoVolume` and `pseudoEchoVolume`?

Reply:
I think that is the official terminus

Reply:
I think that is the official terminology from the SDK and this effect isn't a real echo, so pseudo echo seems more appropriate to me.